.gitlab/e2e/e2e.yml

```diff
@@ -274,6 +274,7 @@ new-e2e-cws:
     - !reference [.manual]
   needs:
     - deploy_deb_testing-a7_x64
+    - deploy_windows_testing-a7
```

Comment:
it would be great to split those into different jobs at some points, the `deploy_windows` jobs are done much later in the pipeline I believe